smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/Configuration.java

```diff
@@ -473,6 +480,16 @@ public Builder setCompatibilityMode(String compatibilityModeString) {
             return setCompatibilityMode(compatibilityMode);
         }
 
+        public Builder setTestRunResultProcessors(String testRunResultProcessorsString) {
+            if (testRunResultProcessorsString == null) {
```

Comment:
The default-value logic shouldn't be hidden in a method called `setTestRunResultProcessors`, see also comment below.